CHANGELOG/v1.6.0.md

```diff
@@ -0,0 +1,294 @@
+- Devtools: Improve Performance dashboard (#9387)
+- Devtools: Make dev cluster networking configurable (#9183)
+- Devtools: Makefile: run doctoc only once (#9182)
+- Devtools: Move tilt-prepare and log-push to tools/internal (#9020)
+- Devtools: Observability: move metrics to config and use sidecar in kube-state-metrics (#9390)
+- Devtools: Refactor docker-push* Makefile targets so users can control with ALL_DOCKER_BUILD which images are pushed (#8586)
+- Devtools: Tiltfile: rename deploy_kustomizations to additional_kustomizations (#9439)
+- e2e: Add back flavor to Waiter interface (#9166)
+- e2e: Add CRS re-reconcile to ownerReference test (#9296)
+- e2e: Add log level for kube components patch to ClusterClass (#9493)
+- e2e: Add MachinePools to Topology Quickstart E2E Templates (#9393)
+- e2e: Add MP timeouts to cluster upgrade tests (#9723)
+- e2e: Add test for ownerReference apiVersion update (#9269)
+- e2e: Add test for scale testing machinery (#9510)
+- e2e: Bump autoscaler to v1.28.0 (#9349)
+- e2e: Drop PR-Informing test tag and job description (#9362)
+- e2e: Dump all pods in e2e test clusters (#9441)
+- e2e: Dump all resource information for self-hosted tests (#9547)
+- e2e: Ensure finalizers are resilient on reconciliation (#9471)
+- e2e: Fail tests if test env version check fails (#9388)
+- e2e: Fix broken e2e test clusterclass (#9506)
+- e2e: Improve labels/annotations in CAPD test ClusterClass (#9469)
+- e2e: Improve logging for self-hosted e2e test (#9637)
+- e2e: Improve output of exec.KubectlApply (#9761)
+- e2e: Refactor e2e ownerRef test utils (#9313)
+- e2e: Test/conformance: add variables to overwrite parralelization of kubetest runs (#9667)
+- e2e: Test/e2e: structure resources by namespace/kind again (#9462)
+- e2e: Use existing value of `SKIP_RESOURCE_CLEANUP` if set in environment (#9152)
+- IPAM: Add age also to v1beta1 IPAM types (#9729)
+- IPAM: Add age column to kubectl output (#9521)
+- IPAM: Enforce IPAM import alias (#9730)
+- KCP: Controlplane: add a test case for syncMachines where the InfraMachine does not exist. (#8992)
+- KCP: Remove disableInPlacePropagation field in KCP controller (#9099)
+- KCP: Remove redundant GetRESTConfig in KCP Management.GetWorkloadCluster (#9448)
+- KCP: Support admin config for Kubeadm v1.29 (#9682)
+- Logging: Change default log level to 2 (#9093)
+- Logging: Fix patch errors not being logged (#9224)
+- Logging: Set controller-runtime logger for clusterctl (#9107)
+- MachinePool: Add MachinePool Builders (#9346)
+- MachinePool: Add util function to get MachinePool by label (#9219)
+- MachinePool: Use SSA Patch to create machines in MP controller (#9802)
+- Metrics: Hack/observability: add capi_machine_status_certificatesexpirydate metric (#9084)
+- Metrics: Hack: bump kube-state-metrics and prometheus charts (#9352)
+- Release: Add additional blocks to release note generation (#9247)
+- Release: Add adilGhaffarDev to release-team owner alias (#9718)
```

Comment:
Side note: I wonder if need a separate category for release, testing, etc. since there are so many "other" changes

Reply:
Also dependency